packages/firebase/index.d.ts

```diff
@@ -768,6 +770,17 @@ declare namespace firebase.firestore {
      * use Timestamp now and opt-in to this new behavior as soon as you can.
      */
     timestampsInSnapshots?: boolean;
+
+    /**
+     * An approximate cache size threshold for the on-disk data. If the cache grows beyond this
+     * size, Firestore will start removing data that hasn't been recently used. The size is not a
+     * guarantee that the cache will stay below that size, only that if the cache exceeds the given
+     * size, cleanup will be attempted.
+     *
+     * The default value is 40 MB. The threshold must be set to at least 1 MB, and can be set to
+     * CACHE_SIZE_UNLIMITED to disable garbage collection.
+     */
+    cacheSizeBytes?: number;
```

Comment:
Can you please also update the externs file?

Reply:
Done.